teamscale-gradle-plugin/src/main/kotlin/com/teamscale/TeamscalePlugin.kt

```diff
@@ -33,10 +33,7 @@ open class TeamscalePlugin : Plugin<Project> {
     }
 
     /** The version of the teamscale gradle plugin and impacted-tests-executor.  */
-    private var pluginVersion = BuildVersion.buildVersion
-
-    /** The version of the teamscale jacoco agent.  */
-    private var agentVersion = BuildVersion.agentVersion
+    private var pluginVersion = BuildVersion.pluginVersion
```

Comment:
why is this a `var`? would `val` not suffice?

Reply:
You are right `val` is sufficient. Changed